documentation/operators/src/nodes/nym-api.md

```diff
@@ -126,17 +126,69 @@ Initialise your API instance with:
 
 You can optionally pass a local identifier for this instance with the `--id` flag. Otherwise the ID of your instance defaults to `default`.
 
+### Enabling credential signing on your Nym API instance
+
+To engage in the Distributed Key Generation (DKG) ceremony, it's essential to transition your `nym-api` instance from its default caching mode to the active credential signing mode. This section guides you through the process of enabling credential signing
+
+#### Generate a new wallet
+
+Begin by generating a new wallet address specifically for your instance to use in credential signing mode. Utilize the `nyxd` command-line tool with the following command:
+
+```
+nyxd keys add signer
+```
+
+It's critical to securely back up the mnemonic phrase generated during this process. This mnemonic is your key to recovering the wallet in the future, so store it in a secure, offline location.
+
+
+#### Fund the address
+
+Next, deposit NYM tokens into the newly created wallet address to ensure it can cover transaction fees incurred during the credential signing process. `nym-api` will not operate if the wallet's balance falls below 10 NYM tokens, displaying an error message upon startup.
+
+We recommand beginning with an initial deposit of 100 NYM tokens and monitoring the balance regularly, topping it up as necessary to maintain operational readiness.
```

Comment:
Do we have an estimate on the token burn rate? E.g. from tests 10 tokens would last X days/weeks?

Reply:
This is heavily dependent on the credential signing volume so don't think we can accurately estimate this.

Reply:
Gotcha, nw